src/components/TodoList.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type Props = {
+  todos: Todo[],
+  deleteHandler: (todoId: number) => void,
+  completeHandler: (todoId: number) => void,
+  editHandler: (todoId: number, newTitle: string) => void,
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  deleteHandler,
+  completeHandler,
+  editHandler,
+}) => {
+  return (
+    <>
+      <ul className="todo-list" data-cy="todosList">
+        {todos.map(todo => (
+          <TodoItem
```

Comment:
1. No need for fragment here
2. Keep `ul` and `li`s in one component. Render TodoItem inside `li`.